### PR TITLE
Add permission and new API in log-manager

### PR DESCRIPTION
**Pull request by Binyang2014**
- Add permission check in log-manage
- Add log-manager API to retrieve log

### CODE REVIEW

File: src/log-manager/config/log-manager.yaml [hunk 1]
```diff
@@ -18,3 +18,7 @@
 service_type: "k8s"
 
 port: 9103
+admin_name: "admin"
+admin_password: "admin"
+jwt_secret: "jwt_secret"
+token_expired_time: 120
```
Comment (hzy46) [inline src/log-manager/config/log-manager.yaml:24]: Does it means `token_expired_second`? I think `token_expired_time` is a bit confusing for admin. He/she may not know the real meaning of it.

File: src/log-manager/config/log-manager.yaml [hunk 2]
```diff
@@ -18,3 +18,7 @@
 service_type: "k8s"
 
 port: 9103
+admin_name: "admin"
```
Comment (hzy46) [inline src/log-manager/config/log-manager.yaml:21]: Please add this change to https://github.com/microsoft/pai/blob/master/contrib/kubespray/quick-start/services-configuration.yaml.template#L239

File: src/log-manager/src/cleaner/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+set -o errexit
+set -o pipefail
+
+log_exist_time=30 # 30 day
```
Comment (hzy46) [inline src/log-manager/src/cleaner/entrypoint.sh:23]: So a job's log will be removed after 30 days? Currently we don't clean user's job in database. If a user views his previous job after it completes for 30 days, and the job log is cleaned, what will happen?

Comment (Binyang2014) [reply src/log-manager/src/cleaner/entrypoint.sh:23]: They can not retrieve the job log. 404 will return. If they want to persistent job log, we recommend customer use external storage, such as AzureBlob or NFS (It's next step, we don't implement yet).

Comment (hzy46) [reply src/log-manager/src/cleaner/entrypoint.sh:23]: If applicable, I think it's better to remind users that their logs will be removed after 30 days in the webportal. It's strange that a previous existing log just disappears and report a 404 error.

Comment (Binyang2014) [reply src/log-manager/src/cleaner/entrypoint.sh:23]: It's the same behavior with previous version. We could add doc for this. For webportal change, it's another enhance and we could open another issue to track

File: src/log-manager/config/log-manager.yaml (hunk 1 above)
Comment (Binyang2014) [reply src/log-manager/config/log-manager.yaml]: Fxied

File: src/log-manager/config/log-manager.yaml (hunk 2 above)
Comment (Binyang2014) [reply src/log-manager/config/log-manager.yaml:21]: Fixed

File: src/log-manager/src/cleaner/entrypoint.sh (hunk 3 above)
Comment (hzy46) [reply src/log-manager/src/cleaner/entrypoint.sh:23]: Agreed. Please open an issue for it.

Resolved this conversation.

**Review by hzy46 — APPROVED** [review]

**Pull request merged**